### ISSUE TITLE
extension.json incorrect setting causing unrecoverable error in extension dialog

### ISSUE THREAD

**Issue by Gizmonuts** [eirannejad/pyRevit#553]
**Describe the bug**
Unrecoverable Error caused by user incorrectly applying setting in extension.json

**To Reproduce**
1. Create a local extension
2. Create the extension.json (file attached below)
3. Set the "author_profile": ""  
4. Open Revit
5. Open pyRevit Extensions dialog
6. Click on the new extension that should cause an error captured by pyRevit
7. Click on another extension in the list and that should cause an unrecoverable error

**Expected behavior**
The initial error should be handled either not requiring the link or providing a prompt to the user to correct.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Win10
 - pyRevit Version [4.7.1-368:2018.3]
 - pyRevit Environment: 
```
==> Installed Revits
Autodesk Revit 2017.2.4 | Version: 17.0.1158.0 | Build: 20181011_1645(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2017"
Autodesk Revit 2018.3.2 | Version: 18.3.2.7 | Build: 20181011_1500(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018"
Autodesk Revit 2019.2 (Update) | Version: 19.2.0.65 | Build: 20181217_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019"
==> Running Revit Instances
PID: 9332 | Autodesk Revit 2018.3.2 | Version: 18.3.2.7 | Build: 20181011_1500(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2018"
==> User Environment
Microsoft Windows 10 [Version 10.0.16299]
Executing User: AURECON\Russ.Strange
Active User: AURECON\Russ.Strange
Adming Access: No
%APPDATA%: "C:\Users\russ.strange\AppData\Roaming"
Latest Installed .Net Framework: "4.7.1"
Installed .Net Target Packs: v3.5 v4.0 v4.5 v4.5.1 v4.5.2 v4.6 v4.6.1 v4.X
pyRevit CLI 0.9.7.0
```

**Additional context**

```json
{
    "builtin": "False",
    "default_enabled": "True",
    "type": "extension",
    "rocket_mode_compatible": "True",
    "name": "test",
    "description": "Testing",
    "author": "bug",
    "author_profile": "",
    "url": "",
    "website": "",
    "image": "",
    "templates": {
        "author": "bug",
        "docpath": ""
    },
    "dependencies": []
}
```

**Comment by eirannejad** [comment]
@Gizmonuts Thanks for reporting this.

**Comment by eirannejad** [comment]
Resolved and will be pushed with 4.6.24
